ppr-ui/src/components/mhrTransfers/TransferType.vue

```diff
@@ -64,6 +64,7 @@
                   class="copy-normal gray7"
                   v-bind="props"
                   :title="null"
+                  @click="handleTypeChange(item.raw)"
```

Comment:
Nice, so this fixes the issue with not firing sometimes?

Reply:
It did in my testing, yes. 
It only seemed to miss firing when the selection was made on the very edges of the element because the event was bound to the item-title, not the item itself. 
So hopefully this resolves whatever associated issues were happening here!